packages/mira/Dockerfile

```diff
@@ -11,10 +11,24 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	software-properties-common
 RUN add-apt-repository ppa:deadsnakes/ppa
 RUN apt-get update && apt-get install -y --no-install-recommends \
-	python3.10 \
-	wget \
-	python3-pip && \
-	rm -rf /var/lib/apt/lists/*
+ python3.10 \
+ python3.10-venv \
```

Comment:
Why funman is running on Python 3.8 but mira 3.10?

Reply:
no idea. but some of the sub images they use 3.8

Reply:
One of Funman's staggered docker mages happens to use ubuntu-20 which defaults to python 3.8. we can probably upgrade if it is desired so long as the C-bindings the numeric libraries still work.